pandas-stubs/core/series.pyi

```diff
@@ -1739,6 +1739,8 @@ class TimestampSeries(Series[Timestamp]):
     # ignore needed because of mypy
     @property
     def dt(self) -> TimestampProperties: ...  # type: ignore[override]
+    @overload
+    def __truediv__(self, other) -> TimestampSeries: ...  # type: ignore
```

Comment:
Why are there no types for other here?

Reply:
> Why are there no types for other here?

i had to add, i thought to complete it after writing tests for np.timedelta64